scripts/measures/lib/validate-change-requests.ts

```diff
@@ -66,7 +68,20 @@ interface Quality_MeasuresChange extends baseMeasuresChange {
     icdImpacted?: string[],
 };
 
-export type MeasuresChange = IA_MeasuresChange | PI_MeasuresChange | Cost_MeasuresChange | Quality_MeasuresChange;
+interface Quality_MeasuresChange extends Base_Quality_MeasuresChange {
+    measureSets?: string[],
+};
+
+interface QCDR_MeasuresChange extends Base_Quality_MeasuresChange {
+    allowedVendors?: string[],
+};
+
+export type MeasuresChange =
+    IA_MeasuresChange &
+    PI_MeasuresChange &
+    Cost_MeasuresChange &
+    Quality_MeasuresChange &
+    QCDR_MeasuresChange;
```

Comment:
Shoulda been an Intersection instead of a Union all along (these terms mean essentially the opposite of the Set Theory terms).